changes to template for revised "check repo" instructions, including …

dwicinas opened the pull request:
…variations for Debian and Ubuntu

Changed base template file and base template file for Debian and Ubuntu

dwicinas commented:
@drothery-edb Take a look at this when you get a chance. I think it's taking shape. I'm not completely happy with the layout of the text for the new information, but I spent a long time trying to get a better look with no luck. I'm still tweaking PEM to ensure that my changes fits smoothly into the existing information. I may not finish that today.

dwicinas commented:
@drothery-edb I think this PR is looking good now. I reviewed changes for all products and they seem to be in order. One caveat is that I think DJ suggested that the command to check on the existence of a repo for SLES is not working. I'm waiting to hear back from him. @nidhibhammar, I made some changes to PEM server to fit in this new verbiage about the repo. I think everything looks good, but you might want to check PEM server to be sure.

nidhibhammar commented on product_docs/docs/pem/9/installing/linux_ppc64le/pem_sles_12.mdx:
```diff
@@ -20,14 +20,19 @@ Before you begin the installation process:
+  To determine if your repository exists, enter this command:
 
-  - To set up the EDB repository, go to [EDB repositories](https://www.enterprisedb.com/repos-downloads) and follow the instructions provided there.
+  `dnf repolist | grep enterprisedb`
```
In SLES `dnf` command doesn't work. It is `zypper`

nidhibhammar commented on product_docs/docs/pem/9/installing/linux_ppc64le/pem_sles_15.mdx:
```diff
@@ -20,14 +20,19 @@ Before you begin the installation process:
+  To determine if your repository exists, enter this command:
 
-  - To set up the EDB repository, go to [EDB repositories](https://www.enterprisedb.com/repos-downloads) and follow the instructions provided there.
+  `dnf repolist | grep enterprisedb`
```
Same comment as above

nidhibhammar commented on product_docs/docs/pem/9/installing/linux_ppc64le/pem_sles_15.mdx:
```diff
@@ -20,14 +20,19 @@ Before you begin the installation process:
+  Alternatively, you can set up the PostgreSQL community repository instead of the EDB repository. To set up the EDB PostgreSQL community repository, go to [yum](https://www.postgresql.org/download/linux/redhat/) or [apt](https://wiki.postgresql.org/wiki/Apt).
 
 - Install the Postgres server. See [Installing EDB Postgres Advanced Server on Linux](/epas/latest/installing/) or [Installing PostgreSQL](/supported-open-source/postgresql/installer/).
 
```
In CentOS7 command is `yum` not `dnf`

nidhibhammar commented on product_docs/docs/pem/9/installing/linux_x86_64/pem_centos_7.mdx:
```diff
@@ -16,14 +16,19 @@ Before you begin the installation process:
 - Set up the repository
 
   Setting up the repository is a one-time task. If you have already set up your repository, you do not need to perform this step.
```
In CentOS7 command is `yum` not `dnf`

nidhibhammar commented on product_docs/docs/pem/9/installing/linux_x86_64/pem_centos_7.mdx:
```diff
@@ -16,14 +16,19 @@ Before you begin the installation process:
+  To determine if your repository exists, enter this command:
 
-  - To set up the EDB repository, go to [EDB repositories](https://www.enterprisedb.com/repos-downloads) and follow the instructions provided there.
+  `dnf repolist | grep enterprisedb`
```
In CentOS7 command is `yum` not `dnf`

djw-m replied:
dnf ships with Centos7, it just isn't default. Yum is also present. Centos8 sees Dnf as default (and for all, going forward).

nidhibhammar commented on product_docs/docs/pem/9/installing/linux_x86_64/pem_centos_7.mdx:
```diff
@@ -16,14 +16,19 @@ Before you begin the installation process:
+  `dnf repolist | grep enterprisedb`
 
-  - To set up the PostgreSQL community repository, go to [yum](https://www.postgresql.org/download/linux/redhat/) or [apt](https://wiki.postgresql.org/wiki/Apt).
+  To set up the EDB EDB repository:
```
Need to remove extra `EDB` from the template

nidhibhammar commented on product_docs/docs/pem/9/installing/linux_x86_64/pem_debian_10.mdx:
```diff
@@ -16,14 +16,19 @@ Before you begin the installation process:
+  `apt-cache search enterprisedb`
 
-  - To set up the PostgreSQL community repository, go to [yum](https://www.postgresql.org/download/linux/redhat/) or [apt](https://wiki.postgresql.org/wiki/Apt).
+  To set up the EDB EDB repository:
```
same comment as above

nidhibhammar commented on product_docs/docs/pem/9/installing/linux_x86_64/pem_debian_11.mdx:
```diff
@@ -16,14 +16,19 @@ Before you begin the installation process:
+  `apt-cache search enterprisedb`
 
-  - To set up the PostgreSQL community repository, go to [yum](https://www.postgresql.org/download/linux/redhat/) or [apt](https://wiki.postgresql.org/wiki/Apt).
+  To set up the EDB EDB repository:
```
same comment as above

nidhibhammar commented on product_docs/docs/pem/9/installing/linux_x86_64/pem_other_linux_8.mdx:
```diff
@@ -17,14 +17,19 @@ Before you begin the installation process:
+  `dnf repolist | grep enterprisedb`
 
-  - To set up the PostgreSQL community repository, go to [yum](https://www.postgresql.org/download/linux/redhat/) or [apt](https://wiki.postgresql.org/wiki/Apt).
+  To set up the EDB EDB repository:
```
Same comment as above

nidhibhammar commented on product_docs/docs/pem/9/installing/linux_x86_64/pem_rhel_8.mdx:
```diff
@@ -16,14 +16,19 @@ Before you begin the installation process:
+  `dnf repolist | grep enterprisedb`
 
-  - To set up the PostgreSQL community repository, go to [yum](https://www.postgresql.org/download/linux/redhat/) or [apt](https://wiki.postgresql.org/wiki/Apt).
+  To set up the EDB EDB repository:
```
Same comment as above

nidhibhammar commented on product_docs/docs/pem/9/installing/linux_x86_64/pem_sles_12.mdx:
```diff
@@ -20,14 +20,19 @@ Before you begin the installation process:
+  To determine if your repository exists, enter this command:
 
-  - To set up the EDB repository, go to [EDB repositories](https://www.enterprisedb.com/repos-downloads) and follow the instructions provided there.
+  `dnf repolist | grep enterprisedb`
```
Command is `zypper` not `dnf`

nidhibhammar commented on product_docs/docs/pem/9/installing/linux_x86_64/pem_sles_12.mdx:
```diff
@@ -20,14 +20,19 @@ Before you begin the installation process:
+  `dnf repolist | grep enterprisedb`
 
-  - To set up the PostgreSQL community repository, go to [yum](https://www.postgresql.org/download/linux/redhat/) or [apt](https://wiki.postgresql.org/wiki/Apt).
+  To set up the EDB EDB repository:
```
Remove extra `EDB` from template

nidhibhammar commented on product_docs/docs/pem/9/installing/linux_x86_64/pem_sles_15.mdx:
```diff
@@ -20,14 +20,19 @@ Before you begin the installation process:
+  To determine if your repository exists, enter this command:
 
-  - To set up the EDB repository, go to [EDB repositories](https://www.enterprisedb.com/repos-downloads) and follow the instructions provided there.
+  `dnf repolist | grep enterprisedb`
```
Command `zypper` not `dnf`

nidhibhammar commented on product_docs/docs/pem/9/installing/linux_x86_64/pem_sles_15.mdx:
```diff
@@ -20,14 +20,19 @@ Before you begin the installation process:
+  `dnf repolist | grep enterprisedb`
 
-  - To set up the PostgreSQL community repository, go to [yum](https://www.postgresql.org/download/linux/redhat/) or [apt](https://wiki.postgresql.org/wiki/Apt).
+  To set up the EDB EDB repository:
```
Remove extra `EDB`

nidhibhammar commented on product_docs/docs/pem/9/installing/linux_x86_64/pem_ubuntu_18.mdx:
```diff
@@ -16,14 +16,19 @@ Before you begin the installation process:
+  `apt-cache search enterprisedb`
 
-  - To set up the PostgreSQL community repository, go to [yum](https://www.postgresql.org/download/linux/redhat/) or [apt](https://wiki.postgresql.org/wiki/Apt).
+  To set up the EDB EDB repository:
```
Remove extra `EDB`

nidhibhammar commented on product_docs/docs/pem/9/installing/linux_x86_64/pem_ubuntu_20.mdx:
```diff
@@ -16,14 +16,19 @@ Before you begin the installation process:
+  `apt-cache search enterprisedb`
 
-  - To set up the PostgreSQL community repository, go to [yum](https://www.postgresql.org/download/linux/redhat/) or [apt](https://wiki.postgresql.org/wiki/Apt).
+  To set up the EDB EDB repository:
```
Remove extra `EDB`

nidhibhammar commented:
@dwicinas, I think we need to change base template file for SLES and Centos 7 as well

dwicinas commented:
@drothery-edb Nidhi raised an issue about the command for CentOS. Now I'm waiting to hear back from DJ as to whether the check repo command for CentOS is okay. So this PR is not quite complete.

dwicinas commented:
@nidhibhammar, I fixed the duplicate "EDB EDB". I also added the correct command for SLES throughout. DJ tells me that the dnf command will work for CentOS 7 so for the sake of simplicity I'm not changing anything for CentOS 7. (BTW, DJ says that while the package manager command for CentOS 7 is yum, CentOS 7 also supports dnf.)

dwicinas commented:
@drothery-edb I think this PR is finally ready for review and hopefully a merge.

djw-m reviewed (commented):
I'm noticing you are telling them to run the command but not how to interpret the output.

Generally "If no output is generated, the repository isn't installed. Follow the instructions to install the repository".

dwicinas commented:
@drothery-edb Made the changes you requested
